apps/server/src/modules/group/controller/mapper/group-response.mapper.ts

```diff
@@ -80,4 +81,14 @@ export class GroupResponseMapper {
 
 		return mapped;
 	}
+
+	static mapToGroupListResponse(groups: Page<ResolvedGroupDto>): GroupListResponse {
+		const mapped: GroupResponse[] = groups.data.map(
+			(group: ResolvedGroupDto): GroupResponse => this.mapToGroupResponse(group)
+		);
+
+		const response: GroupListResponse = { data: mapped, total: groups.total };
```

Comment:
new GroupListResponse(mapped, total, skip, limit) and don't use mapped as a name